File: app/src/main/java/com/dlka/lastfmnewtracknotifier/Date.java

package com.dlka.lastfmnewtracknotifier;

import com.squareup.moshi.Json;

public class Date {

    @Json(name = "uts")
    private String uts;
    @Json(name = "#text")
    private String text;

    public String getUts() {
        return uts;
    }

    public void setUts(String uts) {
        this.uts = uts;
    }

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }

}

File: app/src/main/java/com/dlka/lastfmnewtracknotifier/Date.kt
package com.dlka.lastfmnewtracknotifier

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class Date (
    @Json(name = "uts")
    var uts: String? = null,
    @Json(name = "#text")
    var text: String? = null
)
